code/modules/food_and_drinks/food/foods/ingredients.dm

```diff
@@ -53,6 +53,11 @@
 	filling_color = "#FFF700"
 	tastes = list("cheese" = 1)
 
+/obj/item/food/snacks/cheesewedge/checkpass(passflag)
+	if(passflag & PASSDOOR && ismouse(pulledby))
```

Comment:
Lets just add parenthesis around it for safety because sometimes bit operations and "in" list checkings are dumb and need explicit parentehesis
```suggestion
	if((passflag & PASSDOOR) && ismouse(pulledby))
```